test/LibManagement.cpp

```diff
@@ -167,6 +167,49 @@ TEST_CASE("Valid manifests processing", "[Plugin]") {
     }
 }
 
+TEST_CASE("Valid download caching", "[Plugin]") {
+    smce::Toolchain tc{SMCE_PATH};
+    REQUIRE(!tc.check_suitable_environment());
+
+    const auto cached_downloads_path = SMCE_PATH "/cached_downloads/";
+
+    if (std::filesystem::exists(cached_downloads_path)) {
+        [[maybe_unused]] std::error_code ec_1;
+        std::filesystem::remove_all(cached_downloads_path, ec_1);
```

Comment:
Why is this called `ec_1` and not `ec` ?

Reply:
wont it have conflicts with last one?
should i also be doing a ``REQUIRE_FALSE`` here as well?

Reply:
It's scoped; no conflits.
No `REQUIRE_FALSE` because here we don't rely on the return value

Reply:
i fix